plugins/outputs/opentelemetry/opentelemetry.go

```diff
@@ -173,7 +173,7 @@ func (o *OpenTelemetry) Write(metrics []telegraf.Metric) error {
 const (
 	defaultServiceAddress = "localhost:4317"
 	defaultTimeout        = config.Duration(5 * time.Second)
-	defaultCompression    = "gzip"
+	defaultCompression    = "none"
```

Comment:
Keeping this to gzip so we are more performant by default would be preferred. This would be a breaking change for people who are using this already.

If we do want to change the default, the sample config in the code and README are now out of date as well.

Reply:
You're right. I fixed the "wider compatibility" distraction and reverted this default value.